rocksdb/rocksdb.nim

```diff
@@ -465,6 +465,22 @@ proc releaseSnapshot*(db: RocksDbRef, snapshot: SnapshotRef) =
     rocksdb_release_snapshot(db.cPtr, snapshot.cPtr)
     snapshot.setClosed()
 
+proc flush*(db: RocksDbRef, cfs: openArray[ColFamilyHandleRef]): RocksDBResult[void] =
+  withLock(db.lock):
```

Comment:
The c rocksdb type is thread safe so we shouldn't need to lock here as we don't update any fields in the `RocksDbRef`.